### ISSUE TITLE
Hitbox implementation

### ISSUE THREAD

**Issue by bofhgr** [Legitamine/metanthropes-system#133]

**Comment by bofhgr** [comment]
depends on #158 to figure out the schema

**Comment by bofhgr** [comment]
non-critical, can be deferred to later stages when non-humanoid combatants will be introduced (meta arenas?)

**Comment by bofhgr** [comment]
This will be part of Targeting v2
